### PR TITLE
Hyperparameters change and inference bug fixing

### PR DESCRIPTION
**Pull request by tgjjj**
1. Changed weight decay for finetuning from 5e-4 to 0, and doubled learning rate to 1e-4.
2. Changed output image format from jpg to png.

### CODE REVIEW

**Comment by tgjjj** [comment]
Canceled output image format change.

**Comment by tgjjj** [comment]
Add bug fixing in inference_align.py that causes the output image to be slightly different from that of test.py.
Details:
- Changed warp of reference image to simple padding.
- Fixed many problems about boundary conditions like the width is 128 while the coordinates should range from 0-127.
